### DISCUSSION TITLE
Something wrong in php8.2

### DISCUSSION THREAD

**Discussion by wrangster** [geodesicsolutions-community/geocore-community#210] [Q&A] [answered]
Hi everybody, and hi again to all who's left from the old Geodesic (and Kiwi) forums. Hope you are all good.
I found a problem when trying to change admin password from the default one. It gave me this error message:
Error: Non-static method geoPC::get_hashed_password() cannot be called statically in /home/2/s/*********/www/*****/admin/admin_authentication_class.php:474
Stack trace:
#0 /home/2/s/*********/www/*****/admin/admin_authentication_class.php(519): Admin_auth->update_admin_login(Object(DataAccess), Array)
#1 /home/2/s/*********/www/*****/admin/php5_classes/Admin.class.php(475): Admin_auth->update_admin_tools_password()
#2 /home/2/s/*********/www/*****/admin/index.php(16): geoAdmin->load_page()
#3 {main}

**Comment by wrangster** [comment]
I think I might have found a solution. After trying several ways of sending the request non-statically with no success, I tried to change this:

$password_hashed = geoPC::get_hashed_password($info['username'], $info['password'], $hash_type);

To this:

$password_hashed = geoPC::getInstance();
$password_hashed->get_hashed_password($info['username'], $info['password'], $hash_type);

It seems to work, my password is no changed. I would still love to know if this was the right solution, or if I may have done something awfully stupid. :)

**Comment by wrangster** [comment, marked as answer]
Yeah, I thought I had fixed it, but today it was not possible to log in... :D And as I dont know how to decrypt passwords, I had to just drop in the 4 folders and do the setup once again. Back to square one :).
Luckily someone very nice from here sent me an email this morning, with a solution that really works. (this timeI have tested to log out and in again many times after p.word change :D) What he or she sent was this:

$password_hashed = geoPC::getInstance()->get_hashed_password($info['username'], $info['password'], $hash_type);

So thank you very much, who ever sent me the mail! (No name was in the mail) :)

**Reply by 1kiwi1** [reply]
your welcome ;)

**Reply by wrangster** [reply]
Ahh, who else! Thanks again, 1Kiwi1, and nice to see you again! :)
